src/test/java/org/kcctl/IntegrationTest.java

```diff
@@ -88,6 +90,19 @@ public void cleanup() {
         kafkaConnect.deleteAllConnectors();
     }
 
+    @AfterAll
+    public static void resetKafkaConnectImage() {
+        // In case a test overrode the Kafka Connect image
+        kafkaConnect = kafkaConnectBase;
+    }
+
+    protected void registerTestConnectors(String name, String... names) {
```

Comment:
If this approach looks good, I can file a follow-up to adapt it wherever applicable across the rest of our integration tests.

Reply:
Yeah!

Reply:
Cool! As soon as this is merged I'll throw up a PR.

Reply:
Just kidding, published https://github.com/kcctl/kcctl/pull/354